### PR TITLE
fix(archivedrecordings): use notifications to update state without query

### LINKED ISSUE #344: Archived recordings table view does not update on notification
There is a WebSocket Notification emitted when a recording is added or removed from the archives. This should trigger the Archived recordings table to update and add/remove the corresponding entry. A full refresh (GET query) should not be necessary - the state starts from a query when the view is first created, so just applying add/remove operations in sequence when received by notifications should be enough to keep the view in sync with the actual backend state.

### PR DESCRIPTION
**Pull request by andrewazores**
Fixes #344

### CODE REVIEW

File: src/app/Shared/Services/NotificationChannel.service.tsx [hunk 1]
```diff
@@ -93,24 +92,24 @@ export class NotificationChannel {
...
+    this.messages(NotificationCategory.RecordingArchived).subscribe(v => {
       const event: RecordingNotificationEvent = v.message;
-      notifications.success('Recording Stopped', `${event.recording} was stopped`);
+      notifications.success('Recording Archived', `${event.recording.name} was archived`);
     });
 
     this.messages(NotificationCategory.RecordingDeleted).subscribe(v => {
       const event: RecordingNotificationEvent = v.message;
-      notifications.success('Recording Deleted', `${event.recording} was deleted`);
+      notifications.success('Recording Deleted', `${event.recording.name} was deleted`);
```
Comment (jan-law) [inline src/app/Shared/Services/NotificationChannel.service.tsx:112]: Deleting archived recordings displays the correct file name in the notification, but when I delete a running/stopped recording, the notification reads `undefined was deleted`.

**Review by jan-law — COMMENTED** [review]
Besides my last comment, code looks good

File: src/app/Shared/Services/NotificationChannel.service.tsx (hunk 1 above)
Comment (andrewazores) [reply src/app/Shared/Services/NotificationChannel.service.tsx]: fixed

Comment (jan-law) [reply src/app/Shared/Services/NotificationChannel.service.tsx]: Deleting the active recordings is working well now. When I tried to delete an archived recording, I got a notification reading `[object Object] was deleted`.

Comment (andrewazores) [reply src/app/Shared/Services/NotificationChannel.service.tsx:118]: Ah. Turns out that deleting an active or an archived recording would each trigger a notification with the same category, but (for now - see also #368) different message contents format. I have updated this PR and the corresponding backend one to use more specific categories for these notifications, and to disambiguate the archived/active deletion.

**Review by hareetd — APPROVED** [review]
Tested it out with the corresponding backend and the updated notification categories are looking good. Archived recordings state is updating properly as well.

**Review by jan-law — APPROVED** [review]

**Pull request merged**